tests/cupy_tests/lib_tests/test_strided_tricks.py

```diff
@@ -25,10 +25,45 @@ def test_as_strided(self):
     def test_rolling_window(self, xp):
         a = testing.shaped_arange((3, 4), xp)
         a_rolling = rolling_window(a, 2, 0)
-
         return a_rolling
 
 
+class TestSlidingWindowView(unittest.TestCase):
+    @testing.numpy_cupy_array_equal()
+    def test_1d(self, xp):
+        arr = testing.shaped_arange((3, 4), xp)
+        window_size = 2
+        arr_view = xp.lib.stride_tricks.sliding_window_view(
+            arr, window_size, 0)
```

Comment:
Could you also check the compatibility of `arr_view.strides`?

Reply:
Hello @asi1024 , Can you please clarify what you mean with "compatibility with `arr_view.stride`"? The `arr_view.strides` work fine when I try to access it but I was not sure what you meant.

Please feel free to push anything you think is appropriate for this.